tdrs-backend/tdpservice/parsers/schema_defs/ssp/m1.py

```diff
@@ -15,7 +15,7 @@
         generate_hashes_func=generate_t1_t4_hashes,
         get_partial_hash_members_func=get_t1_t4_partial_hash_members,
         preparsing_validators=[
-            category1.recordHasLengthBetween(113, 150),
```

Comment:
Reading the ticket I don't understand how this change will resolve the issue mentioned in the ticket

Reply:
i believe characters between 113 and 150 would all be empty spaces to make the records even lengths in the file. if i'm wrong on that, i have the same question as you. my understanding is also that we don't care about extra "junk" data submitted on the end of the line, spaces or not. so checking max length is no longer relevant.